src/diffbot.js

```diff
@@ -209,7 +79,8 @@ class Diffbot {
    * @returns {Object} The query results
    */
   knowledgeGraph(options) {
-    let diffbot_url = `https://kg.diffbot.com/kg/dql_endpoint?token=${this.token}&query=${encodeURIComponent(options.query)}`;
+    console.log(options);
+    let diffbot_url = `https://kg.diffbot.com/kg/dql_endpoint?token=${this.token}&query=${encodeURIComponent(options.query)}&size=1000`;
```

Comment:
No need to hardcode size in here, it's an option you can pass
```suggestion
    let diffbot_url = `https://kg.diffbot.com/kg/dql_endpoint?token=${this.token}&query=${encodeURIComponent(options.query)}`;
```